File: 2011. Final Value of Variable After Performing Operations/2011. Final Value of Variable After Performing Operations.js
/**
 * @param {string[]} operations
 * @return {number}
 */
var finalValueAfterOperations = function(operations) {
    var X=0
    var n=operations.length
    for(let i=0; i<n;i++){
        if(operations[i] == "X++" || operations[i] == "++X"){
            X++;
        }
        else{
            X--;
        }
    }
    return X    
};

File: 2011. Final Value of Variable After Performing Operations/2011. Final Value of Variable After Performing Operations.c
int finalValueAfterOperations(char** operations, int operationsSize) {
    int count=0;
    for(int i=0; i<operationsSize; i++){
        if(strcmp(operations[i], "X++")==0 || strcmp(operations[i], "++X")==0){
            count+=1;
        }else{
            count-=1;
        }        
    }
    return count;    
}
